### ISSUE TITLE
Directives are joined with "," instead of " " in 2.0

### ISSUE THREAD

**Issue by ulrichb** [frux/csp-header#23]
In 2.0 The following doesn't work anymore:

```JS
{
  directives: {
  // ...
  "connect-src": ["'self'", oidcAuthorityUrl, "https://translation.googleapis.com"],
  // ...
  }
}
```
because the rules are string-joined with a ",".

.. instead I have to use ```"connect-src": [`'self' ${oidcAuthority} https://translation.googleapis.com`],``` (note the spaces).

Is this change on purpose?

**Comment by frux** [comment]
Hi! No it's obviously my fault. Will fix ASAP.

**Comment by frux** [comment]
@ulrichb just have fixed it, added tests for this case and published `2.0.2` with fix included. Thank you for the report and sorry for inconvenience.
NPM RunKit: https://runkit.com/embed/lfpl7lxfcndf

**Comment by ulrichb** [comment]
Cool. Thx!
